src/couch_replicator_doc_processor.erl

```diff
@@ -239,7 +242,8 @@ worker_returned(Ref, Id, {temporary_error, Reason}) ->
             state = error,
             info = Reason,
             errcnt = ErrCnt + 1,
-            worker = nil
+            worker = nil,
+            last_updated = os:timestamp()
```

Comment:
Ah good point. Let's pull that up into worker_return then. All cases of update_docs_row update it so it can be done in one place only